### ISSUE TITLE
Error in documentation

### ISSUE THREAD

**Issue by jhallard** [fchollet/keras#1235]
In the documentation on this (http://keras.io/callbacks/)  page for callback, it says that the key values for the logs dictionary sent to the `on_epoch_end` function will contain 

  * val_loss
  * val_accuracy

The key is actually *val_acc*. Likewise, for the `on_batch_end` function, it claims the keys will be

  * loss
  * accuracy

Once again, the real keys are *loss* and *acc*. Took me a little while to get this sorted out.

**Comment by fchollet** [comment]
Right. This is now fixed.

**Comment by jhallard** [comment]
Thank you kind person!
